pkg/cmd/cluster/cluster.go

```diff
@@ -9,9 +9,10 @@ import (
 
 func NewCmdCluster(f *factory.Factory) *cobra.Command {
 	cmd := cobra.Command{
-		Use:  "cluster <command>",
-		Args: cobra.ArbitraryArgs,
-		Long: "View cluster information",
+		Use:   "cluster <command>",
+		Args:  cobra.ArbitraryArgs,
+		Long:  "View cluster information",
+		Short: "View cluster and queue information",
```

Comment:
Haha its funny the short version is longer than the long. Should they be switched around? Or maybe just the same thing? (queue might end up being a separate command at this stage so perhaps shouldnt include that in the cluster description)

Reply:
Yes it is :D , fixed it now to be the same